src/postgres/tree/PostgresFunctionTreeItem.ts

```diff
@@ -36,4 +39,12 @@ export class PostgresFunctionTreeItem extends AzureTreeItem<ISubscriptionContext
     public get iconPath(): TreeItemIconPath {
         return getThemeAgnosticIconPath('Collection.svg');
     }
+
+    public async update(newDefinition: string): Promise<string> {
```

Comment:
Would prefer to have this code directly in `PostgresFunctionEditor.ts`. A whole lot of code is _related_ to tree items, but shouldn't be put in the class itself because it would get out of hand quickly